nvflare/private/fed/server/server_app_runner.py

```diff
@@ -50,7 +54,20 @@ def sync_up_parents_process(self, args, server):
         server.engine.create_parent_connection(int(args.conn))
         server.engine.sync_clients_from_main_process()
 
-    def set_up_run_config(self, server, conf):
+    def set_up_run_config(self, workspace: Workspace, server, conf):
+        runner_config = conf.runner_config
+        assert isinstance(runner_config, ServerRunnerConfig)
```

Comment:
should this check be done inside `ServerJsonConfigurator` ?

Reply:
Will address this in another PR